### ISSUE TITLE
After keras update, there is an exception, "Regularizers cannot be reused"

### ISSUE THREAD

**Issue by summerHaha** [fchollet/keras#4561]
code is as follows:
TimeDistributed(Dense(5, activation='softmax', activity_regularizer=activity_l1(0.01)))(lstm)

I look at the documentation and see how it should be used, but do not understand why it throws an exception.

**Comment by joelthchao** [comment]
This works well, can you provide code snippet that can reproduce error?
```python
from keras.models import Model
from keras.layers import Input, Dense, TimeDistributed
from keras.regularizers import activity_l1

input = Input(shape=(3, 3))
result = TimeDistributed(Dense(5, activation='softmax', activity_regularizer=activity_l1(0.01))) (input)
m = Model([input], [result])
m.compile(optimizer='sgd', loss='mse')
```
FYI, exception comes from [here](https://github.com/fchollet/keras/blob/master/keras/regularizers.py#L35)

**Comment by fchollet** [comment]
Exception is self-evident: instantiate a new regularizer every time rather than reusing the same instance twice. This will be refactored in the near future, though.
